### ISSUE TITLE
Question: Is masking the training data needed?

### ISSUE THREAD

**Issue by cpitman** [MathiasGruber/PConv-Keras#13]
Not sure if it would be much of an optimization, but I noticed the current code produces masked versions of the images for training, and I'm not sure it is a needed step.

Since the partial convolution will use the mask to zero out any contribution from masked pixels, would there be any impact to using the ground truth image? And if it is possible, would there be any benefit?

**Comment by MathiasGruber** [comment]
During my training I wasn't limited by the data processing pipeline, but there is some potential benefit I reckon in not doing that masking. It's definitely not adding anything - I initially just did it to ensure that the network didn't have that information when doing the inpainting - kinda a sanity check thing.
